en/uavcan/README.md

```diff
@@ -1,70 +1,93 @@
+## Wiring
 
-### Wiring
+All UAVCAN components share the same connection architecture/are wired the same way.
+Connect all on-board UAVCAN devices into a chain and make sure the bus is terminated at the end nodes (the order in which the nodes are connected/chained does not matter).
 
-The first step is to connect all UAVCAN enabled devices with the flight controller.
-The following diagram displays how to wire all components.
-The used Zubax devices all support a redundant CAN interface in which the second bus is optional but increases the robustness of the connection.
+The following diagram shows this for a flight controller connected to [UAVCAN motor controllers (ESCs)](../uavcan/escs.html) and a UAVCAN GNSS.
 
 ![UAVCAN Wiring](../../assets/uavcan/uavcan_wiring.png)
 
-It is important to mention that some devices require an external power supply \(e.g. Zubax Orel 20\) and others can be powered by the CAN connection \(e.g Zubax GNSS\) itself.
-Please refer to the documentation of your hardware before continuing with the setup.
+The diagram does not show any power wiring.
+Refer to your manufacturer instructions to confirm whether components require separate power or can be powered from the CAN bus itself. 
+
+For more information about proper bus connections see [UAVCAN Device Interconnection](https://kb.zubax.com/display/MAINKB/UAVCAN+device+interconnection) (Zubax KB).
+
+:::note
+- While the connections are the same, the _connectors_ may differ across devices.
+- An second/redundant" CAN interface may be used, as shown above (CAN2).
+  This is optional, but can increase the robustness of the connection.
+:::
+
+
+## PX4 Configuration
+
+In order to use UAVCAN components with PX4 you will first need to enable the UAVCAN driver:
+
+1. Power the vehicle using the battery (you must power the whole vehicle, not just the flight controller) and connect *QGroundControl*.
+1. Navigate to the **Vehicle Setup > Parameters** screen.
+1. [UAVCAN_ENABLE](../advanced_config/parameter_reference.md#UAVCAN_ENABLE) must be [set](../advanced_config/parameters.md) to one of the non-zero values.
+
+   The values are:
+   - `0`: UAVCAN driver disabled.
+   - `1`: Sensors Manual Config.
+   - `2`: Sensors Automatic Config.
+   - `3`: Sensors and Actuators (ESCs) Automatic Config
+
+   Use `1` if _none_ of the connected UAVCAN devices support automatic configuration (check the manual!), `2` or `3` if _some_ of them support automatic configuration, and `3` if you're using UAVCAN ESCs (this assigns motor controls to the UAVCAN bus rather than PWM).
+
+:::note
+You will need to manually allocate static ids for any nodes that don't support automatic configuration.
+When using dynamic configuration, any manually allocated ids should be given a value greater than the number of UAVCAN devices (to avoid clashes).
+:::
+
+Most UAVCAN sensors require no further setup (they are plug'n'play, unless specifically noted in their documentation).
```

Comment:
@bkueng Is this reasonable? My understanding is that all you have to do is enumerate most sensors and they just work. However there are quite a lot of UAVCAN parameters. So it may be that all you have to do is AUTO enumerate most sensors - if you have things that don't support auto setup perhaps there is more configuration to be done.

Can you clarify this point? Or am I OK just saying this?

Reply:
I think that's accurate and the most important thing (node ID's).